lib/python/rosie/svn_pre_commit.py

```diff
@@ -124,6 +148,22 @@ def run(self, repos, txn):
             if not names[-1]:
                 tail = names.pop()
 
+            # Allow root property modification by super users.
```

Comment:
We typically avoid having any super-users defined so this is going to make updates to the property more awkward. Probably not a big deal since the mapping hopefully won't change (unless some users don't have accounts at the beginning)?

I've realised that we already have a a special suite containing metadata:
https://metomi.github.io/rose/doc/html/installation.html#rosie-special-suite
So, an alternative would be to define the mapping in a file in this special suite.
We could then use the normal suite access mechanisms to allow updates.

Reply:
Agreed, good idea - done